pkgs/tools/misc/fzf/default.nix

```diff
@@ -27,6 +27,11 @@ buildGoPackage rec {
     cp $src/bin/fzf-tmux $bin/bin
     mkdir -p $out/share/vim-plugins
     ln -s $out/share/go/src/github.com/junegunn/fzf $out/share/vim-plugins/${name}
```

Comment:
I don't think we need it at all after multiple outputs. Other packages can use `fzf` source explicitly like https://github.com/NixOS/nixpkgs/pull/18487/files#diff-5274f44419e19a44108863e49e8265e8L104. Did I miss something?